homeassistant/components/roomba/irobot_base.py

```diff
@@ -118,6 +120,16 @@ def battery_stats(self):
         """Return the battery stats."""
         return self.vacuum_state.get("bbchg3", {})
 
+    @property
+    def last_mission(self):
+        """Return last mission start time."""
+        if (
+            ts := self.vacuum_state.get("cleanMissionStatus", {}).get("mssnStrtTm")
+        ) is None or ts == 0:
+            return None
+        tz = ZoneInfo(self.vacuum_state.get("timezone", "UTC"))
```

Comment:
I think you need to use `dt_util.async_get_time_zone` to get a ZoneInfo object. For more info check <https://developers.home-assistant.io/blog/2024/05/19/fix_zoneinfo_blocking_io>

Reply:
Thanks for the feedback. I've updated this line to use `dt_util.async_get_time_zone`.